dts/bindings/sensor/avago,apds9960.yaml

```diff
@@ -14,3 +14,4 @@ properties:
     int-gpios:
       type: phandle-array
       required: true
+      description: Interrupt pin, open drain, active low
```

Comment:
Active level and other pin properties are defined by DTS flags. I propose to change the description to 
```
        Interrupt pin.

        This pin signals active low when produced by the controller. The
        property value should ensure the flags properly describe the signal
        that is presented to the driver.
```
as already used by other bindings.

Reply:
I oppose. What is the information content of your proposal? Much text to say it is active low?

Reply:
Sorry, I owe a more detailed explanation. The usage of the active low property is a bit opaque. I'll copy a comment I provided for another chip in a similar case.

> In principle the flags are there to encode cause -> effect relationship. This is easy to understand when taking button as an example. Regardless of the way the button is designed and the circuitry surrounding it if we attach GPIO_ACTIVE_LOW to the pin it means: if the button is pressed we expect input signal on the pin to be low. If we attach GPIO_ACTIVE_HIGH to the pin it means: if the button is pressed we expect input signal on the pin to be high. The same holds true for MCP2515 chip. The flag is there to tell the driver what happens when MCP2515 asserts its interrupt line. In a typical case the interrupt output of MCP2515 will be wired directly to the GPIO pin on the MCU. This should be described in DTS by adding GPIO_ACTIVE_LOW flag to the pin property. However, it's also legal to attach GPIO_ACTIVE_HIGH flag in case the signal from MCP2515 to MCU is routed on the board via an inverter.

In short the flags are there to adjust the driver to the specific board they work on. We could shorten the description to: _The interrupt pin of APDS9960 is open-drain, active low. If connected directly the MCU pin should be configured as pull-up, active low._

Reply:
While the proposed text from the earliest merged conversions is verbose, it's intended to simplify maintenance (what does the bare device produce) and use by less experienced users (how does the way it's connected to your MCU affect the flags you should be using in dts).  It would be nice to describe GPIO signals consistently, but I agree it would be acceptable to use a shorter description that still shows what is necessary in the devicetree to support an open-drain active-low signal, viz. that a pull-up must be present, either in hardware or by enabling it in the GPIO.

Reply:
@mnkp @pabigot thanks, will fix it